docs/examples/storage/preauthrequests/README.md

```diff
@@ -0,0 +1,31 @@
+    #     ___  ____     _    ____ _     _____
```

Comment:
Favoring single file examples that begin with a comment stating what the example plan demonstrates.